docker-compose-dev-local-images.yaml

```diff
@@ -3,27 +3,52 @@ version: '3'
 #Build only (put your images in here)
 services:
 #An exmaple of how to add your code to databox
-  databox-os-monitor-driver:
+  driver-os-monitor:
     build:
-        context: ./databox-os-monitor-driver
+        context: ./driver-os-monitor
         dockerfile: Dockerfile${DATABOX_ARCH}
```

Comment:
Fixed in 26f72ba2b4dc436b104ed4cb307dde959ac43886

On x86 DATABOX_ARCH == "" so it points to the default Dockerfile adding a separator here breaks this.